tests/TestCase.php

```diff
@@ -9,4 +9,22 @@
 abstract class TestCase extends BaseTestCase
 {
     use CreatesApplication;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->disableConfigExternalApis();
+    }
+
+    protected function tearDown(): void
+    {
+        $this->disableConfigExternalApis();
```

Comment:
Is this needed? I would expect it only to be needed inside the `setUp` method

Reply:
It felt safer but it's probably not needed indeed

Reply:
Removed in [516f319](https://github.com/cultuurnet/publiq-platform/pull/28/commits/516f3197a562b455b637be8e08b665cdca2c2017)